src/Analysis/Ast/Impl/Extensions/AstExtensions.cs

```diff
@@ -60,9 +62,20 @@ public static Expression FindExpression(this PythonAst ast, SourceLocation locat
             return ast.CommentLocations[match].Column < location.Column;
         }
 
+        internal static bool HasComment(this PythonAst ast, IPythonModule module, string comment, int lineNum) {
```

Comment:
This is unlikely to be used anywhere except linter since it is not particularly common to look for a specific comment. So this is basically a function somewhere inside linter reporting. `GetLineComment` might be useful though.

Reply:
I misinterpreted what you said above.